### PR TITLE
Add Bearer.com to companies that <3 ViewComponent

### PR DESCRIPTION
**Pull request by yshmarov**
Add Bearer.com to list of companies that use ViewComponent :)

### Summary

Add Bearer.com to list of companies that use ViewComponent

### Other Information

We also write a bit about our experience with ViewComponent https://www.bearer.com/blog/snapshot-testing-viewcomponents-with-rspec
:)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -203,6 +203,7 @@ ViewComponent is built by over a hundred members of the community, including:
 * [Shogun](https://getshogun.com/)
 * [Wecasa](https://www.wecasa.fr/)
 * [Wrapbook](https://wrapbook.com/)
+* [Bearer](https://www.bearer.com/) (70+ components)
```
[the remark this answers is not in the document]
Comment (yshmarov) [reply docs/index.md]: Sure! Updated :)

**Review by Spone — APPROVED** [review]

**Comment by Spone** [comment]
Can you add a changelog entry?

**Review by Spone — APPROVED** [review]

**Comment by Spone** [comment]
Thanks! You just need to resolve the conflict and I'll merge!

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ title: Changelog
 
 ## main
 
+## 2.53.0
+
```
Comment (joelhawksley) [inline docs/CHANGELOG.md:11]:
```suggestion
```

**Review by joelhawksley — APPROVED** [review]

**Pull request merged**